jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/listGroup/DisabledItemsTests.java

```diff
@@ -59,10 +60,15 @@ public void listGroupTextTests(int num, String text) {
     }
 
     @Test(dataProvider = "listData")
-    public void listGroupClassesIsValidationTests(int num, String text) {
-        listGroupDisabledItems.listGroup.get(num).is()
+    public void listGroupClassesTextValidationTests(int num, String text) {
+        Assert.assertEquals(listGroupDisabledItems.listGroup.get(num).getText(), text);
```

Comment:
Lets use JDI asserts in tests for bootstrap instead of bare testNG's:
listGroupDisabledItems.listGroup.get(num).is().text(text);